apps/api-catalogue/screens/ServiceList/ServiceList.tsx

```diff
@@ -30,45 +23,44 @@ import {
   Query,
   QueryGetApiCatalogueArgs,
 } from '@island.is/api/schema'
+import {
+  PricingCategory,
+  DataCategory,
+  TypeCategory,
+  AccessCategory,
+} from '@island.is/api-catalogue/consts'
+import { style } from 'treat'
 
 interface PropTypes {
   top?: ReactNode
   left: ReactNode
   right?: ReactNode
   bottom?: ReactNode
-  className?: string
-  listClassNames?: string
+  isMobile: boolean
 }
 
-function ServiceLayout({
-  top,
-  left,
-  right,
-  bottom,
-  className,
-  listClassNames,
-}: PropTypes) {
+function ServiceLayout({ top, left, right, isMobile }: PropTypes) {
```

Comment:
Feel like the layout should be in a seperate component and imported instead of written here.

Reply:
I agree, but we are focusing on moving the FilterSearch and InputSearch into the shared library.  Later, when we integrate the filter into the web, we should tackle this issue.

Reply:
I Agree, as I noted above we should go over this file and ServiceList.treat.ts together to merge our changes.